hooks/post/30-upgrade.rb

```diff
@@ -31,13 +31,16 @@ def import_subscriptions
 end
 
 def elasticsearch_message
-  return true unless Kafo::Helpers.execute('rpm -q elasticsearch')
-
-  rpms = ['ruby193-rubygem-tire', 'tfm-rubygem-tire', 'elasticsearch', 'sigar-java', 'sigar', 'snappy-java', 'lucene4-contrib', 'lucene4']
-  message = "Elasticsearch has been removed as a dependency.  The database files can be "\
+  `rpm -q elasticsearch`
+  if $?.success?
+    rpms = ['ruby193-rubygem-tire', 'tfm-rubygem-tire', 'elasticsearch', 'sigar-java', 'sigar', 'snappy-java', 'lucene4-contrib', 'lucene4']
+    message = "Elasticsearch has been removed as a dependency.  The database files can be "\
             "removed manually with # rm -rf /var/lib/elasticsearch.  "
-  message += "Some packages are no longer needed and can be removed:  # yum erase #{rpms.join(' ')}"
-  Kafo::Helpers.log_and_say :info, message
+    message += "Some packages are no longer needed and can be removed:  # yum erase #{rpms.join(' ')}"
+    Kafo::Helpers.log_and_say :info, message
+  else
+    logger.info "Elasticsearch already removed, skipping"
```

Comment:
We get a duplicate message when the installer runs, here is the code:

  else
    Kafo::Helpers.log_and_say :info, "Elasticsearch already removed, skipping"

Here is the output:

Upgrade Step: elasticsearch_message...
[ INFO 2017-02-06 09:15:43 verbose] Upgrade Step: elasticsearch_message...
Elasticsearch already removed, skipping
[ INFO 2017-02-06 09:15:43 verbose] Elasticsearch already removed, skipping

Wanting to keep it logger.info to keep the noise down in the logs/terminal.